app/Http/Controllers/DashboardController.php

```diff
@@ -3,16 +3,73 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\DailyMenu;
+use App\Material;
+use App\Order;
+use App\User;
+use Carbon\Carbon;
 
 class DashboardController extends Controller
 {
     /**
+     * Current date
+     *
+     * @var string
+     */
+    protected $curDate;
+
+    /**
+     * The User implementation.
+     *
+     * @var User
+     */
+    protected $user;
+
+    /**
      * Create a new controller instance.
      *
+     * @param Carbon\Carbon $carbon carbon instance
+     * @param User          $user   user instance
+     *
      * @return void
      */
-    public function __construct()
+    public function __construct(Carbon $carbon, User $user)
+    {
+        $carbon = $carbon->today();
+        $format = $carbon->format('Y-m-d');
+        $this->curDate = $format;
+        $this->user = $user;
+    }
+
+    /**
+     * Get total of new order, register, ... on day
+     *
+     * @param string $modelType type of model
+     *
+     * @return int
+     */
+    public function getTotalOf($modelType)
```

Comment:
You can create a `trait` call `OrderStatistic` and create a function `getTotalOnDate`. To make flexible you can use a param for date. `getTotalOnDate($date = $curentDate)` by default $date is curent date, but when you want to get co another date, you can change param `$date`;